src/borg/archiver/prune_cmd.py

```diff
@@ -9,7 +9,15 @@
 from ..archive import Archive
 from ..cache import Cache
 from ..constants import *  # NOQA
-from ..helpers import ArchiveFormatter, interval, sig_int, ProgressIndicatorPercent, CommandError, Error
+from ..helpers import (
+    ArchiveFormatter,
+    interval,
+    int_or_interval,
+    sig_int,
+    ProgressIndicatorPercent,
+    CommandError,
+    Error,
+)
 from ..helpers import archivename_validator
```

Comment:
could you please move `interval, int_or_interval` to the next import line, so it doesn't make that multi-line import from it?

Reply:
I didn't even notice there were two lines at all! Shouldn't they all be on the same line and let the linter deal with it?

Reply:
I don't really like that vertical arrangement, thus I try to stay below the limit.